exporter/azure_monitor/common/common.go

```diff
@@ -3,27 +3,25 @@ package common
 
 import (
 	"context" 
+	"runtime"
+	"os"
+	"fmt"

```

Comment:
sort

Reply:
Good catch. I will sort all the imports alphabetically in my files.